html/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/AttrsScope.kt

```diff
@@ -246,3 +246,10 @@ open class AttrsScopeBuilder<TElement : Element>(
 private val setClassList: (HTMLElement, Array<out String>) -> Unit = { e, classList ->
     e.classList.add(*classList)
 }
+
+/**
+ * Adds a boolean attribute.
+ * @see AttrsScope.attr
+ */
+fun AttrsScope<*>.attr(attr: String, value: Boolean = true) =
+    attr(attr, value.toString())
```

Comment:
Any specific reason making it an extension function?
What do you think about adding this function to the `AttrsScope` itself?

Would you like to add a test for this new function in AttributesTests?

Reply:
Thanks for replying. Adding it as a member function looks good to me too. And sure I'd love to add some tests for this.